Classes/Custom/Realurl.php

```diff
@@ -141,6 +133,26 @@ public function addAutoConfig( $params, &$pObj )
 	}
 
 	/**
+	 * Adds no_cache to configuration if not already defined
+	 *
+	 * @param array $configuration
+	 * @return void
+	 */
+	protected function addNoCache(array &$configuration) {
+		if ( array_search( 'no_cache', $configuration ) !== false ) {
```

Comment:
That won't work because `array_search()` doesn't search in multi-dimensional arrays